data/pigui/modules/map-sector-view.lua

```diff
@@ -120,33 +121,38 @@ local function showSearch()
 	ui.text(lc.SEARCH)
 	search_text, changed = ui.inputText("", search_text, {})
 	if search_text ~= "" then
-		local systempaths = Engine.SearchNearbyStarSystemsByName(search_text)
-		if #systempaths == 0 then
-			ui.text(lc.NOT_FOUND)
+		local parsedSystem = SystemPath.ParseString(search_text)
+		if parsedSystem ~= nil then
+			Engine.SectorMapGotoSectorPath(parsedSystem)
 		else
-			local data = {}
-			for _,path in pairs(systempaths) do
-				local jumpStatus, distance, fuelRequired, duration = getHyperspaceDetails(path)
-				table.insert(data, { jumpStatus = jumpStatus, distance = distance, fuelRequired = fuelRequired, duration = duration, path = path })
-			end
-			ui.child("search_results", function ()
-								 table.sort(data, function(a,b)
-								 							if a.path == b.path or a.path:IsSameSystem(b.path) then
-																return false
-															end
-								 							return a.distance < b.distance
-								 end)
-								 for _,item in pairs(data) do
-									 local system = item.path:GetStarSystem()
-									 local label = system.name
-									 label = label .. '  ' .. item.jumpStatus .. ", " .. string.format("%.2f", item.distance) .. lc.UNIT_LY .. ", " .. item.fuelRequired .. lc.UNIT_TONNES .. ", " .. ui.Format.Duration(item.duration, 2)
+			local systempaths = Engine.SearchNearbyStarSystemsByName(search_text)
+			if #systempaths == 0 then
+				ui.text(lc.NOT_FOUND)
+			else
+				local data = {}
+				for _,path in pairs(systempaths) do
+					local jumpStatus, distance, fuelRequired, duration = getHyperspaceDetails(path)
+					table.insert(data, { jumpStatus = jumpStatus, distance = distance, fuelRequired = fuelRequired, duration = duration, path = path })
+				end
+				ui.child("search_results", function ()
+									 table.sort(data, function(a,b)
+																if a.path == b.path or a.path:IsSameSystem(b.path) then
+																	return false
+																end
+																return a.distance < b.distance
+									 end)
+									 for _,item in pairs(data) do
+										 local system = item.path:GetStarSystem()
+										 local label = system.name
+										 label = label .. '  ' .. item.jumpStatus .. ", " .. string.format("%.2f", item.distance) .. lc.UNIT_LY .. ", " .. item.fuelRequired .. lc.UNIT_TONNES .. ", " .. ui.Format.Duration(item.duration, 2)
 
-									 if ui.selectable(label, false, {}) then
-										 Engine.SetSectorMapSelected(item.path)
-										 Engine.SectorMapGotoSystemPath(item.path)
+										 if ui.selectable(label, false, {}) then
+											 Engine.SetSectorMapSelected(item.path)
+											 Engine.SectorMapGotoSystemPath(item.path)
+										 end
 									 end
-								 end
-			end)
+				end)
+			end
 		end
```

Comment:
Is indentation correct here? I could imagine everything is too indented in the inner block,  or something? the `end` don't seem equidistant in indentation. I could be wrong. Just voicing concern.

Reply:
There is a block that's more indented that the rest, but that seems common throughout the file and I just indented it by an extra tab with this PR.
It seems to follow the indentation set by the parantheses that wrap it.

Reply:
OK.